frontend/routes/Profile.vue

```diff
@@ -0,0 +1,248 @@
+<template>
+<div style="position: relative;">
+    <nav>
+        <a href="#"><div class="icon"></div></a>
+        <a href="#">Logout</a>
+    </nav>
+
+    <header>
+        <div class="banner"></div>
+        <div class="icon"></div>
+
+        <h1 class="username">John Smith <a class="edit"></a></h1>
```

Comment:
what's this `a`?

Reply:
This would eventually be replaced with a `<router-link>` to a profile edit view. `<a>` is the closest tag we can use in the meantime, since they're eventually the same element.

I'll add a `href="#"` so that it actually looks clickable for now.